chromadb/test/property/test_embeddings.py

```diff
@@ -525,378 +525,533 @@ def test_embeddings_flake1(client: ClientAPI) -> None:
     state = EmbeddingStateMachine(client)
```

Comment:
nit (unrelated to your PR): the name of the test does not convey anything meaningful

Reply:
that ship has sailed, sunk and been under submarine obeservation for a while now :)